.all-contributorsrc

```diff
@@ -192,6 +192,13 @@
         "bug",
         "code"
       ]
+    },
+    {
+      "login": "lukaszmoroz",
+      "name": "Łukasz Moroz",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/1408542?v=3",
+      "profile": "https://github.com/lukaszmoroz",
+      "contributions": []
```

Comment:
This should probably include code and test. Feel free to update if you like. Thanks!